views/js/qtiCreator/plugins/menu/preview.js

```diff
@@ -138,11 +144,21 @@ define([
                         cssClass: 'preview-trigger',
                         testId: 'preview-the-item'
                     })
-                ).on('click', e => previewHandler(e, this, id));
+                ).on('click', e => previewHandler(e, uri, this, id));
             };
 
-            //creates the preview button
-            this.elements = config.providers ? config.providers.map(createButton) : [createButton()];
+            /* creates the preview buttons 
+            in some cases there are more preview buttons than one, that can happen if there are different preview providers
+            or different items to be previewed (when we are editing item translation in creator)*/
+            if(translation) {
+                this.elements = [
+                    createButton({uri: originalItemUri, label: 'Preview original', title: 'Preview original item'}),
+                    createButton({uri: translatedItemUri, label: 'Preview translation', title: 'Preview item translation'})
+                ]
+            }else{
+                //if configured with multiple preview providers, will show button for each
+                this.elements = config.providers ? config.providers.map(createButton) : [createButton()];
+            }
```

Comment:
**quibble:** the code does no respect the common style, did you activate eslint and prettier?